syntax = "proto3";

package portfolio;

import "models/util/local_timestamp.proto";
import "models/util/uuid.proto";

option java_multiple_files = true;
option java_package = "common.models.portfolio";
option java_outer_classname = "PortfolioProtos";

message PortfolioProto {
  //1-4 reserved
  string object_class = 1;
  string version = 2;

  //Primary Key
  util.UUIDProto uuid = 5;
  util.LocalTimestampProto as_of = 6;
  bool is_link = 7;

  string portfolio_name = 10;
}
